_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -198,78 +176,78 @@ Autocomplete is inappropriate for the type of field.
 
 #### Inapplicable Example 1
 
-Inapplicable element.
+This `button` element does not support `autocomplete` attributes.
 
 ```html
 <button autocomplete="username"></button>
 ```
 
 #### Inapplicable Example 2
 
-Autocomplete attribute is empty ("").
+This `autocomplete` [attribute value][] is empty ("").
 
 ```html
 <label>Username<input autocomplete=""/></label>
 ```
 
 #### Inapplicable Example 3
 
-The element is hidden through `display:none`.
+This `autocomplete` [attribute value][] contains only [ASCII whitespace][].
 
 ```html
-<label>Username<input autocomplete="username" style="display:none"/></label>
+<label>Username<input autocomplete=" "/></label>
 ```
 
 #### Inapplicable Example 4
 
-The `input` element has a `type` attribute that is in the `button` state.
+This `autocomplete` [attribute value][] is on an element is hidden through `display:none`.
 
 ```html
-<label>Username<input type="button" autocomplete="username"/></label>
+<label>Username<input autocomplete="username" style="display:none"/></label>
 ```
 
 #### Inapplicable Example 5
 
-The `input` element has a `type` attribute that is in the `hidden` state.
+This `autocomplete` attribute is on an `input` element with a `type` [attribute value][] that does not support autocomplete.
 
 ```html
-<label>Username<input type="hidden" autocomplete="username"/></label>
+<label>Username<input type="button" autocomplete="username"/></label>
 ```
 
 #### Inapplicable Example 6
 
-The `input` element has an HTML `disabled` attribute.
+This `autocomplete` attribute is on an `input` element that has the `disabled` attribute.
 
 ```html
 <label>Username<input autocomplete="username" disabled/></label>
 ```
```

Comment:
I'm actually not seeing anything in Applicability about the `disabled` attribute 🤔 
Well, I guess that setting `disabled` will also automatically set `aria-disabled`… But then this description needs a bit more details (and the example is similar to the next). Otherwise, the Applicability needs to update the **disabled** condition to take that into account, probably referencing https://act-rules.github.io/glossary/#disabled-element

Reply:
I noticed that too. I'm tackling that separately.